src/Helper/SortHelper.php

```diff
@@ -4,14 +4,16 @@
 
 namespace Webgriffe\SyliusElasticsearchPlugin\Helper;
 
+use Sylius\Component\Core\Model\TaxonInterface;
+
 final class SortHelper implements SortHelperInterface
 {
     public function retrieveSearchSorting(array $sortingQueryParams = []): array
     {
         return $sortingQueryParams; // If empty it will sort by _score desc as default
     }
 
-    public function retrieveTaxonSorting(array $sortingQueryParams = []): array
+    public function retrieveTaxonSorting(array $sortingQueryParams = [], ?TaxonInterface $taxon = null): array
```

Comment:
```suggestion
    public function retrieveTaxonSorting(array $sortingQueryParams = [], TaxonInterface $taxon): array
```
We are in 0.x releases.. So you can still introduce BC